pkg/types/installconfig.go

```diff
@@ -154,6 +154,9 @@ type InstallConfig struct {
 	// BootstrapInPlace is the configuration for installing a single node
 	// with bootstrap in place installation.
 	BootstrapInPlace *BootstrapInPlace `json:"bootstrapInPlace,omitempty"`
+
+	// Capabilities configures the installation of optional, core cluster components
+	Capabilities *Capabilities `json:"capabilities,omitempty"`
```

Comment:
Should this be optional? Are we going to omit the capabilities configuration from the ClusterVersion resource when this is omitted? I would expect that we would still set a default value in the ClusterVersion resource.

Reply:
Yes. This should be optional. I've added the macro comment.

Reply:
OK. Optional does make sense. But I do think we want to populate with the default, even when the entire `Capabilities` has been omitted. I am also leaning towards **if** the user provided any values for `Include` or `Exclude`, then they are also required to make an explicit decision about `InclusionDefault` (despite it being redundant with whether they chose `Include` or `Exclude`).

Reply:
The design has changed since the enhancement and I've updated the PR with the new ideas. PTAL